addons/point_of_sale/static/src/app/screens/payment_screen/payment_screen.js

```diff
@@ -233,7 +233,7 @@ export class PaymentScreen extends Component {
             this.hardwareProxy.openCashbox();
         }
 
-        this.currentOrder.initialize_validation_date();
+        this.currentOrder.initialize_date_order();
```

Comment:
can we change this to some meaningful name?
we have already initialized the `date_order` in the setup method of the Order `PosModel` and from here we are updating this already initialized `date_order`

Reply:
Ok sir,
should i use "update_date_order".